lib/KeyboardAwareScrollView.js

```diff
@@ -20,6 +20,14 @@ const KeyboardAwareScrollView = React.createClass({
     this.setResetScrollToCoords(this.props.resetScrollToCoords)
   },
 
+  scrollTo: function (...args) {
+    return this.refs._rnkasv_keyboardView.scrollTo(...args)
```

Comment:
There's a built-in function in the mixin: `scrollToPosition` https://github.com/APSL/react-native-keyboard-aware-scroll-view/blob/master/lib/KeyboardAwareMixin.js#L107

This is not necessary.